person.rb

```diff
@@ -0,0 +1,23 @@
+class Person
+  # Constructor
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id
+
+  attr_accessor :name, age
```

Comment:
* Your `age` is missing a `:` kindly add it so that you can write and read the age instance variable